tests/algorithms/test_mce_irl.py

```diff
@@ -26,16 +26,17 @@ def rollouts(env, n=10, seed=None):
     rv = []
     for i in range(n):
         done = False
+        # if a seed is given, then we use the same seed each time (should
+        # give same trajectory each time)
+        obs = env.reset(seed=seed)
         if seed is not None:
-            # if a seed is given, then we use the same seed each time (should
-            # give same trajectory each time)
-            env.seed(seed)
             env.action_space.seed(seed)
-        obs = env.reset()
+
         traj = [obs]
         while not done:
             act = env.action_space.sample()
-            obs, rew, done, info = env.step(act)
+            obs, rew, terminated, truncated, info = env.step(act)
```

Comment:
Only `obs` and `rew` seem to be used here. Thoughts on updating to:
```
obs, rew, _, _, _ = env.step(act)
traj.append((obs, rew))
```
?

Reply:
`terminated` and `truncated` are merged to `done`, which again is used as the termination condition for the loop. I however made sure the info is ignored.